src/ServiceManagement/Compute/ComputeManagement/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Virtual Machines and Hosted Services.")]
 
-[assembly: AssemblyVersion("13.0.0.0")]
-[assembly: AssemblyFileVersion("13.0.0.0")]
+[assembly: AssemblyVersion("14.0.0.0")]
```

Comment:
I am not sure about what you meant by "API surface". I added a new command "initiateMaintenance" and I also added a new field into the response of GetDeployment. Does that count as "new API surface"?